### DISCUSSION TITLE
Broot

### DISCUSSION THREAD

**Discussion by mimame** [catppuccin/catppuccin#1848] [Port Requests]
https://dystroy.org/broot/
**Are you available to help in the creation of this port? 😄 (optional)**
Maybe (I can help but I'm a bit colorblind)

**Additional context 📷 (optional)**
Add any other context or screenshots about the port request here.

**Comment by aandriamgit** [comment]
bruh

**Comment by sgoudham** [comment]
Closing as any further discussion should take place in #2840
